Python/Product/Debugger/Debugger/PythonProcess.cs

```diff
@@ -1043,16 +1052,14 @@ public PythonProcess(PythonLanguageVersion languageVersion, string exe, string a
                     string mapTo = mappingInfo[toDebuggee ? 1 : 0];
 
                     if (file.StartsWith(mapFrom, StringComparison.OrdinalIgnoreCase)) {
-                        if (file.StartsWith(mapFrom, StringComparison.OrdinalIgnoreCase)) {
-                            int len = mapFrom.Length;
-                            if (!mappingInfo[0].EndsWith("\\")) {
-                                len++;
-                            }
-
-                            string newFile = Path.Combine(mapTo, file.Substring(len));
-                            Debug.WriteLine(String.Format("Filename mapped from {0} to {1}", file, newFile));
-                            return newFile;
+                        int len = mapFrom.Length;
+                        if (!mappingInfo[0].EndsWith("\\")) {
+                            len++;
                         }
+
+                        string newFile = Path.Combine(mapTo, file.Substring(len));

```

Comment:
Just noting here, but this is same code that was already there, there was just a double if statement that I removed.  Unfortunately, it seems to be highlighting the whole block instead of just the duplicate if that was removed.  Your comments are probably still valid though :)